src/utils/probe-utils.js

```diff
@@ -12,14 +10,6 @@ export function sortByKey(key) {
   };
 }
```

Comment:
I know this isn't part of this pull request, so I'm happy to open a follow-up issue about this, but in reading through this file, I noticed something. Should line 7 be `if (a < b)`? If they're equal we should return zero.